.github/workflows/transport-ci.yml

```
@@ -0,0 +1,196 @@
+      - name: Get latest transport version and increment
+        id: next_version
+        run: |
+          # Get the latest transport tag (using transports/ prefix to match docker build)
+          LATEST_TAG=$(git tag -l 'transports/v*' | sort -V | tail -n 1)
+          if [ -z "$LATEST_TAG" ]; then
+            # If no transport tag exists, start with v0.1.0
+            NEW_TAG="transports/v0.1.0"
+          else
+            # Extract version numbers
+            VERSION=${LATEST_TAG#transports/v}
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)